examples/twentyseven-pt-stencil-solver/doc/tooltip

```diff
@@ -0,0 +1 @@
+Use array views to generate stencil matrix. Solve a linear system.
```

Comment:
Mention that it is a 27-pt stencil, here and in the 9pt stencil as well, otherwise it is the same as the 3pt stencil.